### PR TITLE
fix: dispatch initial action to add initial state

### PR DESCRIPTION
**Pull request by BlackFenix2**
I have added a useEffect hook on the StateInspector component to pass a mock-dispatch to the redux store. This will add the intialState from each child useState() and useReducer() component to Redux Dev Tools on page load.

all the local jest tests passed and i confirmed functionality on the example project.

I haven't learned how to handle/override the @@INIT, i'm still looking into that specific scenario.

### CODE REVIEW

**Comment by BlackFenix2** [comment]
@troch  i totally don't mean to rush you, but i am super in love with this devtool.

**Comment by troch** [comment]
Hey, sorry I'm on holiday, will come back to you on this once I'm back :)

File: src/StateInspector.tsx [hunk 1]
```diff
@@ -92,6 +95,11 @@ export const StateInspector: React.FC<StateInspectorProps> = ({
         return store
     }, [])
 
+    useEffect(() => {
+        store && store.dispatch({ type: "REINSPECT/@@INIT", payload: {} })
+        return () => {}
```
[the remark this answers is not in the document]
Comment (BlackFenix2) [reply src/StateInspector.tsx]: got it, i removed the unused unmount function for the useEffect hook.

**Comment by BlackFenix2** [comment]
I removed the function call, are we good to go @troch ?

**Pull request merged**